src/ticketmap/index.tsx

```diff
@@ -151,6 +151,18 @@ const $costRanges = createDeepEqualSelector(
   }
 )
 
+const areSetsEqual = (setA: Set, setB: Set): boolean => {
```

Comment:
`setA: Set` seems a bit redundantly redundant. Perhaps: 

```suggestion
const areSetsEqual = (a: Set, b: Set): boolean => {
```